demo/docbot/ingestion.py

```diff
@@ -39,12 +39,28 @@ def read_files_from_data() -> list:
 
   return result
 
-def ingest_to_opensearch(data):
-  # client = opensearch_connection_builder()
-  # client.bulk()
-  pass
+def ingest_to_opensearch(data) -> int:
+  """
+  Args:
+    data: a list of json documents to be ingested
+  Returns:
+    int: 1 if successfully ingested
+  """
+  client = opensearch_connection_builder()
+  docs = []
+  for point in range(len(data)):
+    docs.append({"index": {"_index": "docbot", "_id": point}})
+    docs.append(json.dumps(data_list[point]))
+
+  response = client.bulk(docs)
+  if response["errors"]:
+    raise ValueError("Failed to insert data into client.")
+  else:
+    print(f"Bulk-inserted {len(response['items'])} items.")
+
+  return 1
```

Comment:
Lets leave out the return here. If an exception is not raised we can assume it ran without error.